.github/workflows/build.yml

```diff
@@ -13,6 +13,7 @@ on:
     - '**/*.rst'
   pull_request:
     branches:
+    - pep585
```

Comment:
Is it needed?

Reply:
Only as long as we're using this branch for the PR. I'll remove it after merging master once last time.